packages/commands/src/utils.mjs

```diff
@@ -75,30 +75,34 @@ export const initializeAudiusLibs = async (handle) => {
 
 let audiusSdk;
 export const initializeAudiusSdk = async ({ apiKey = undefined, apiSecret = undefined } = {}) => {
-  const discoveryNodeSelector = new DiscoveryNodeSelector({
-    healthCheckThresholds: {
-      minVersion: developmentConfig.minVersion,
-      maxBlockDiff: developmentConfig.maxBlockDiff,
-      maxSlotDiffPlays: developmentConfig.maxSlotDiffPlays,
-    },
-    bootstrapServices: developmentConfig.discoveryNodes,
-  })
-  const entityManager = new EntityManager({
-    discoveryNodeSelector,
-    web3ProviderUrl: developmentConfig.web3ProviderUrl,
-    contractAddress: developmentConfig.entityManagerContractAddress,
-    identityServiceUrl: developmentConfig.identityServiceUrl,
-    useDiscoveryRelay: true,
-  })
+
+  const solanaRelay = new SolanaRelay(
+    new Configuration({
+      basePath: '/solana',
+      headers: {
+        'Content-Type': 'application/json'
+      },
+      middleware: [
+        {
+          pre: async (context) => {
+            const endpoint = 'http://audius-protocol-discovery-provider-1'
```

Comment:
nit: Do we use any kind of env var or programmatic generation of this we could leverage? Feels like an easy way to get frustrated if something happens to change or you're running multiple DNs locally.

Reply:
open to ideas - ideally this is temporary until we use relays broadly across nodes

Reply:
Don't have any good recommendations at the moment, just wanted to call it out in case you did! 😄